core/src/main/java/com/exadel/aem/toolkit/api/annotations/policies/AllowedChildren.java

```diff
@@ -44,7 +44,6 @@
      * to any template.
      * <p>You can use wildcards, e.g. {@code "/conf/acme/settings/wcm/*", "*templates/design2/homepage",
      * "*templates/design2/*"}.
-     * <p><u>Note</u>: this setting should not be used to specify static templates.
```

Comment:
this limitation is also mentioned in the md file for allowed components. could you please update that file as well?